Gruntfile.js

```diff
@@ -90,7 +90,7 @@ module.exports = function(grunt) {
         options: {
           // The src/vendor paths are needed to find the CF components' files.
           // Feel free to add additional paths to the array passed to `concat`.
-          paths: grunt.file.expand('src/vendor/*').concat([])
+          paths: '<%= loc.src %>/vendor/'
```

Comment:
Essentially. Before it was creating a comma sep list of every individual cf-component dir. That's not necessary, you only need the root where they live (ie the vendor directory). Making it static like this is both clearer and less error prone.